book/content/remote_events/intro.md

```diff
@@ -0,0 +1,9 @@
+## Prerequisites / recommended skill level
+
+| Prerequisite | Importance | Notes |
+| -------------|----------|------|
+|  |  |  |
+
+## Moving Collaborative Activities Online
+
+## How this will help you/ why this is useful
```

Comment:
```suggestion
## How this will help you/ why this is useful

## Don't do it alone!

Most collaborative online activities will include participants who may not be familiar with the tools you use, or the concept of the activity in general.
Especially in an online setting, it is highly recommendable to have at least one other person who can help participants with setting up the tools correctly, troubleshoot, or take over specific tasks, such as muting participants or monitoring questions or requests to speak.
```